rs/nns/governance/src/proposals/self_describing.rs

```diff
@@ -124,6 +130,46 @@ impl<T: Into<SelfDescribingValue>> From<Option<T>> for SelfDescribingValue {
     }
 }
 
+impl<T: Into<SelfDescribingValue>> From<Vec<T>> for SelfDescribingValue {
```

Comment:
I think this is a good time to use `where`:

```suggestion
impl<T> From<Vec<T>> for SelfDescribingValue
where
    T: Into<SelfDescribingValue>
```

But also, we can avoid `Into`, like so:

```suggestion
impl<T> From<Vec<T>> for SelfDescribingValue
where
    SelfDescribingValue: From<T>
```

I really think this is nicer:

```rust
value.into_iter().map(SelfDescribingValue::from).collect()
```

More explicit.